### PR TITLE
Feature/add build and push image workflow

### PR DESCRIPTION
**Pull request by tuechki**

### CODE REVIEW

File: .github/workflows/build_and_push.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Build and Push Docker Image
```
Comment (Nizamski) [inline .github/workflows/build_and_push.yml:1]: Is this workflow only related to development images?

Comment (tuechki) [reply .github/workflows/build_and_push.yml:1]: No, just a workflow to create and push image manually from whatever branch.

**Review by Nizamski — APPROVED** [review]
